pkg/dbcrypt/config/config.go

```diff
@@ -26,6 +26,9 @@ func Read() (config CryptoConfig, err error) {
 	if err != nil {
 		return config, err
 	}
-	log.Debug().Msgf("CryptoConfig: %+v", config)
+	log.Debug().
```

Comment:
I know the log statement was already there, but is it a good idea to print those sensitive values to the logs ?

Reply:
Most likely not but it's a debug statement which should not be emitted in a production release, and as you mentioned it was already there and the scope of work in this structured logging ticket will be based on the existing statements.

If you want to remove the password field (which I agree with) the `%v` representation should be completely removed, and the `crypto_config_v1_password` field as well.

Reply:
I would assume the customer is free to set the log level. When encountering an error the log level could be set to debug on customer side to create a support package containing (debug) logs (just some speculation on my side, don't know the actual process).

I also don't see much benefit of the log message, and potential for harm. At least I would not be comfortable to have that in the code.